pkg/policy/snapshots.go

```diff
@@ -60,6 +60,10 @@ func RestoreSnapshot(ctx context.Context, conn LowLevelQueryExecer, log hclog.Lo
 		return fmt.Errorf("invalid filename: %q", fileName)
 	}
 	source_name := strings.TrimPrefix(strings.TrimSuffix(fileName, ".csv"), "table_")
+	err = deleteFKs(ctx, conn, log, source_name)
+	if err != nil {
+		return fmt.Errorf("error removing fks from %q: %w", source_name, err)
+	}
```

Comment:
question around this why isn't the data restore order consistent i.e parent -> child ? Then there shouldn't be any FK issue and no need to delete snapshots.

Reply:
The idea around snapshots was to store the minimum amount of data required for the sql query to run